aten/src/ATen/core/ivalue.cpp

```diff
@@ -113,6 +113,16 @@ std::string ivalue::Object::name() const {
   return this->type_->qualname();
 }
 
+IValue ivalue::Object::getAttr(const std::string& name) const {
```

Comment:
Probably want the same api as pybind11, which uses `attr`

Reply:
pybind's attr returns a functor which lets you write to the object by assigning to it. I think that's probably a little overkill in this case.

I think it'll expose static `getattr` and `setattr` like the python builtins

Reply:
The old API seems simpler, this makes it more like a C API. Maybe we shouldn't try to match pythons semantics here?

Reply:
I'm okay with that. whatever is more natural for people I guess